README.md

```diff
@@ -4,6 +4,7 @@ ArcGIS Runtime SDK 100.10 for Qt sample applications.  The repo contains [Qt](ht
 # Prerequisites
 * Qt SDK 5.15 or higher (Open source or commercial)
 * Qt Creator
+* [ArcGIS Runtime Toolkit - Qt](https://github.com/Esri/arcgis-runtime-toolkit-qt) is required as some samples utilize toolkit components. For more information on the toolkit please visit, [Building a better toolkit.](https://community.esri.com/community/developers/native-app-developers/arcgis-runtime-sdk-for-qt/blog/2020/08/28/building-a-better-toolkit)
```

Comment:
i'd leave out the blog link - that's more of a background detail that isn't required for basic setup instructions of the sample repo

```suggestion
* [ArcGIS Runtime Toolkit - Qt](https://github.com/Esri/arcgis-runtime-toolkit-qt) is required as some samples utilize toolkit components. 
```